util/topsql/reporter/reporter.go

```diff
@@ -133,6 +134,18 @@ func NewRemoteTopSQLReporter(client ReportClient) *RemoteTopSQLReporter {
 	return tsr
 }
 
+var (
+	ignoreRegisterSQLCnt          = metrics.TopSQLIgnoredCounter.WithLabelValues("register-sql")
+	ignoreRegisterPlanCnt         = metrics.TopSQLIgnoredCounter.WithLabelValues("register-plan")
+	ignoreCollectCPUTimeCnt       = metrics.TopSQLIgnoredCounter.WithLabelValues("collect-data")
+	ignoreReportDataCnt           = metrics.TopSQLIgnoredCounter.WithLabelValues("report-data")
```

Comment:
I would prefer to use `register_sql` to match other labels:

![image](https://user-images.githubusercontent.com/1916485/124231974-94c28580-db43-11eb-891e-0594506bef07.png)